### PR TITLE
Fixed Face ID and added logging calendar view

### PR DESCRIPTION
**Pull request by andreaslordos**
- Fixed/finalized Face ID login for now

-  Set up FSCalendar, a custom iOS calendar library that allows for much more customization than the regular iOS calendar library. This customization is useful in the logging because it frees up a lot of screen real estate (the default iOS date picker doesn’t have this sort of “week view” that I have above. 

- FSCalendar is as of now, without docs because the website that they hosted their docs on ([cocoadocs.org](http://cocoadocs.org/)) is down, so it took me some extra time to set up some things 

- Customized the calendar a lot to make it look nice and consistent with the rest of the UI, while also building it in such a way that it will be easy to change the appearance in the future without much effort

FSCalendar is definitely going to be very useful when I implement my calendar view of the app, because I’ll want different cells to display different images etc. so today’s hours spent on learning this library were definitely a good investment. Should be done with logging tomorrow

**tl;dr fixed faceid, started logging with custom calendar library**

### CODE REVIEW

File: Hera/ViewControllers/Main Flow/LoggingViewController.m [hunk 1]
```diff
@@ -10,33 +10,51 @@
 @interface LoggingViewController () <FSCalendarDelegate, FSCalendarDelegateAppearance, FSCalendarDataSource>
 
 @property (strong, atomic) NSDate *maxDate;
+@property (strong, atomic) UIColor *cellBgColor;
```
Comment (stanyang95) [inline Hera/ViewControllers/Main Flow/LoggingViewController.m:13]: Instead creating so many UIColors property, we can create UIColor constants to label.

File: Hera/ViewControllers/Main Flow/LoggingViewController.m [hunk 2]
```diff
@@ -50,30 +68,52 @@ - (BOOL)pastOrPresent:(NSDate *)date {
...
+
+
 - (UIColor *)calendar:(FSCalendar *)calendar appearance:(FSCalendarAppearance *)appearance fillDefaultColorForDate:(NSDate *)date {
     if ([self pastOrPresent:date]) {
         // color normally
-        return [UIColor lightGrayColor];
+        return self.cellBgColor;
+    }
+    else {
+        // give disabled color
```
Comment (stanyang95) [inline Hera/ViewControllers/Main Flow/LoggingViewController.m:84]: Can delete since the code says its functionality

File: Hera/ViewControllers/Main Flow/LoggingViewController.m [hunk 3]
```diff
@@ -50,30 +68,52 @@ - (BOOL)pastOrPresent:(NSDate *)date {
...
+- (NSString *)calendar:(FSCalendar *)calendar subtitleForDate:(NSDate *)date {
+    NSDateFormatter *dateFor = [[NSDateFormatter alloc]init];
+    [dateFor setDateFormat:@"MMM"];
+    return [dateFor stringFromDate:date]; // i.e. Jul for July
+}
+
+
 - (UIColor *)calendar:(FSCalendar *)calendar appearance:(FSCalendarAppearance *)appearance fillDefaultColorForDate:(NSDate *)date {
     if ([self pastOrPresent:date]) {
         // color normally
```
Comment (stanyang95) [inline Hera/ViewControllers/Main Flow/LoggingViewController.m:80]: Can also delete

**Review by stanyang95 — APPROVED** [review]
LGTM

**Review by stanyang95 — APPROVED** [review]

File: Hera/ViewControllers/Main Flow/LoggingViewController.m (hunk 1 above)
Comment (andreaslordos) [reply Hera/ViewControllers/Main Flow/LoggingViewController.m:13]: Not entirely how that would work, could you please give an example?

Comment (stanyang95) [reply Hera/ViewControllers/Main Flow/LoggingViewController.m:13]: You can do something like:
`static UIColor *const color = .... `
is possible in Objective-C++. Just change the extension for *.mm

Comment (stanyang95) [reply Hera/ViewControllers/Main Flow/LoggingViewController.m:13]: Some useful link:
https://stackoverflow.com/questions/2824187/objective-c-defining-uicolor-constants
https://stackoverflow.com/questions/2718507/how-do-i-define-constant-values-of-uicolor

Comment (andreaslordos) [reply Hera/ViewControllers/Main Flow/LoggingViewController.m]: Done, thank you! Moved color constants to cc file

**Review by stanyang95 — APPROVED** [review]
Thanks for the changes!
